var/spack/repos/builtin/packages/py-tensorboard-plugin-wit/package.py

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import tempfile
+from spack import *
 
 
 class PyTensorboardPluginWit(Package):
```

Comment:
If this PR takes much longer, I would be interested in splitting this into a separate PR and merging immediately. I'm having build issues with this package on multiple platforms.

Reply:
This should subclass `PythonPackage` now, we now support installing wheels. Then you can remove `extends` and the pip dep